docker-compose.yml

```diff
@@ -7,10 +7,10 @@ volumes:
   broker_var: {}
   coordinator_var: {}
   router_var: {}
-  druid_shared: {}
   postgres_data: {}
   postgres_metadata: {}
   redis_data: {}
+  druid_shared: {}
```

Comment:
Do we need this here for when people run both this and the druid docker-compose file?

Sort of related, when I run `docker compose up` I'm getting:
```
service "dj" depends on undefined service druid_coordinator: invalid compose project
```
We need to remove that from the `dj` depends-on block.

Reply:
After removing the `druid_coordinator` it worked. At first db migration was complaining no matter how many times I removed all volumes and tore everything down. Eventually I just did `docker system prune` and then everything worked. I guess my docker environment was just generally funky.
![docker](https://user-images.githubusercontent.com/43911210/193182448-c1486ec8-b916-4928-86aa-7b7d6f9eff95.jpg)

Reply:
I am not sure how to do this best. For a no-druid setup we don't need the `druid_coordinator` but for the with-druid setup we should have it. I need to try and see if I can redefine the `dj` service component in the `docker-compose-druid.yml` when running `make docker-run-with-druid`...

Reply:
It worked... oh man this PR. I am burned out :)